notebooks/tutorial/1 - Setup.ipynb

```diff
@@ -70,7 +81,7 @@
     "import re\n",
     "import warnings\n",
     "\n",
-    "if not re.match(\"\\S+.\\d{4}-\\d{2}-\\d{2},\\d{4}-\\d{2}-\\d{2}.*\\S*\",header_name):\n",
+    "if not re.match(r\"\\S+.\\d{4}-\\d{2}-\\d{2},\\d{4}-\\d{2}-\\d{2}[.\\S*]*\",header_name):\n",
```

Comment:
Can we export this warn function and use it here from the dsdk?

Reply:
yes, but that requires pulling in dsdk, releasing new version, then upgrading the version in here.

Reply:
Raised as #11 for later